graylog2-web-interface/src/pages/UserCreatePage.tsx

```diff
@@ -23,29 +23,34 @@ import { Button } from 'components/bootstrap';
 import { PageHeader, DocumentTitle } from 'components/common';
 import UserCreate from 'components/users/UserCreate';
 import UsersPageNavigation from 'components/users/navigation/UsersPageNavigation';
+import useProductName from 'customization/useProductName';
 
-const UserCreatePage = () => (
-  <DocumentTitle title="Create New User">
-    <UsersPageNavigation />
-    <PageHeader
-      title="Create New User"
-      actions={
-        <LinkContainer to={Routes.SYSTEM.USERS.CREATE}>
-          <Button bsStyle="success">Create user</Button>
-        </LinkContainer>
-      }
-      documentationLink={{
-        title: 'Permissions documentation',
-        path: DocsHelper.PAGES.USERS_ROLES,
-      }}>
-      <span>
-        Use this page to create new Graylog users. The users and their permissions created here are not limited to the
-        web interface but valid and required for the REST APIs of your Graylog server nodes, too.
-      </span>
-    </PageHeader>
+const UserCreatePage = () => {
+  const productName = useProductName();
 
-    <UserCreate />
-  </DocumentTitle>
-);
+  return (
+    <DocumentTitle title="Create New User">
+      <UsersPageNavigation />
+      <PageHeader
+        title="Create New User"
+        actions={
+          <LinkContainer to={Routes.SYSTEM.USERS.CREATE}>
+            <Button bsStyle="success">Create user</Button>
+          </LinkContainer>
+        }
+        documentationLink={{
+          title: 'Permissions documentation',
+          path: DocsHelper.PAGES.USERS_ROLES,
+        }}>
+        <span>
+          Use this page to create new users. The users and their permissions created here are not limited to the web
```

Comment:
Maybe we can simplify this sentence:

```suggestion
          Use this page to create new users for the web interface or the REST API.
```